src/site/content/en/handbook/grammar/index.md

```diff
@@ -20,10 +20,16 @@ description: |
   }
 </style>
 
-Grammar and mechanics are important for making writing clear, but don't let them get in the way of putting text on the page! Content reviewers will always support you with editorial stuff. That said, here are a few conventions to be aware of.
+Grammar and mechanics are important for making writing clear. Content reviewers review posts and codelabs for editorial issues covered in this post. If a piece is already error free, the entire review process goes faster. To keep things simple, web.dev uses the following resources:
+* The [Google Developer Documentation Style Guide](https://developers.google.com/style/) (GDDSG) is the primary reference for writing style issues.
+* If a style issue isn't covered in the GDDGS, check the [Chicago Manual of Style](https://www.chicagomanualofstyle.org/home.html).
+* [Merriam-Webster](http://www.m-w.com) is the primary reference for spelling and capitalization.
+* The [Word list](/handbook/word-list) provides spelling and capitalization conventions specific to web.dev.
+
+The rest of this post describes grammar styles that are specific to web.dev and are different from the core resources.
```

Comment:
This isn't true at the moment. This list is more of a greatest hits from the core references. But, as we discussed, I think it probably _should_ be true. I'll incorporate it into a larger revision of this post.